f/connectors/comapeo/tests/conftest.py

```diff
@@ -160,6 +160,86 @@ class CoMapeoServer:
     )
 
 
+@pytest.fixture
+def comapeoserver_with_failing_attachments(mocked_responses):
+    """A mock CoMapeo Server with observations that have failing attachments"""
+
+    @dataclass
+    class CoMapeoServer:
```

Comment:
what's this type represent?  It feel its name is probably misleading -- and also overloaded (compare to `comapeo_server(TypedDict)` in `f/connectors/comapeo/comapeo_pull.py`)